src/Integration.UnitTests/MefServices/VsSessionHostTests.cs

```diff
@@ -71,6 +71,20 @@ public void TestInitialize()
 
         #region Tests
 
+        [TestMethod]
+        public void MefCtor_CheckIsExported()
```

Comment:
Adding missing test case. The `VsSessionHost` has always (?) been MEF-created, but we didn't have a test for it.

The `ActiveSolutionBoundTracker` is also MEF-creatable but doesn't have a `MefCtor_CheckIsExported` test. I looked at creating one, but it would have been more complicated since the constructor does other things e.g. fetches services, registers events etc. That test would have required a lot more setup, so I chose not to do it.